src/panels/config/dashboard/ha-config-navigation.ts

```diff
@@ -19,10 +20,16 @@ class HaConfigNavigation extends LitElement {
 
   @property() public pages!: PageNavigation[];
 
+  @property() public externalConfig?: ExternalConfig;
+
   protected render(): TemplateResult {
     return html`
       ${this.pages.map((page) =>
-        canShowPage(this.hass, page)
+        (
+          page.path === "#external-app-configuration"
```

Comment:
?
```suggestion
          page.component === "mobile_app"
```

Reply:
No, your instance have the integration loaded, but this should only show if you are visiting from a app

Reply:
yes, this still does that?